torchlog/torchlog.go

```diff
@@ -41,7 +41,7 @@ func init() {
 func getPrefix(level string, color *color.Color) string {
 	currentTime := time.Now().Format("15:04:05")
 	toColoredString := color.SprintFunc()
-	return toColoredString(fmt.Sprintf("%s[%s]", level, currentTime))
+	return toColoredString(fmt.Sprintf("%s[%s] ", level, currentTime))
```

Comment:
Minor formatting change to make the output a little nicer. The previous output looked like:
```
INFO[19:00:29]Run pprof command: go tool pprof -raw -seconds 30 main.test prof.cpu
INFO[19:00:29]Writing svg to torch.svg
```

I thought adding a space would make the output a little more readable:
```
INFO[19:00:29] Run pprof command: go tool pprof -raw -seconds 30 main.test prof.cpu
INFO[19:00:29] Writing svg to torch.svg
```

Caught this while updating the examples in `README`